### ISSUE TITLE
Small tweaks

### ISSUE THREAD

**Issue by awicenec** [bartTC/tab#1]
Hi Martin, 

thanks for this nice little script. I've tried to use it in one of my projects and found three small little issues:
1) When trying to use the options in the tabfile I got a key error. This is due to the line
setattr(options[0], n[1]) which should read
setattr(options,n[0], n[1])
2) The configparser makes the options (first item on a line) lower case and since the first item on a line is used for the commands, they are all translated to lower case, which can cause issues in some cases. To fix that I've introduced the option to put the commands on lines starting with cmd<n> = where <n> is a running number in each section. The current way still works, but the cmd<n> one is more secure.
3) When opening a new window sometimes the default cd command is executed after the actual user commands from the tabfile. 
If you like I can send you my version of the script and you can modify it. In addition I've also tested it using the configparser module from PyPI in Python-2.7. 

-awicenec

**Comment by bartTC** [comment]
Thank you for the feedback! I did not used a tabfile for a while, there were numerous bugs. Also the tabfile description the README was wrong, I've updated it as well.

Yeah I really like to see your changes. Best is you just fork it and make a pull request.

FWIW I had the Py2.7 configparser in the past, but the problem is that it treats `:` and `=` as a option separater. This breaks a command like `open http://www.example.com/`.
